Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -456,9 +464,16 @@ public async Task ProcessExecutionResult(SubmissionExecutionResult submissionExe
         submission.Processed = true;
         submission.ProcessingComment = null;
 
+        var submissionForProcessing = await this.submissionsForProcessingData.GetBySubmission(submission.Id);
```

Comment:
Here also, why get the entity again? I thinkg the entity is retrieved once in `MarkProcessed` method below. Just save the result in the same transaction that updates it to Processed.

Reply:
Done.